### ISSUE TITLE
venv build fails on macOS Big Sur

### ISSUE THREAD

**Issue by lutzroeder** [onnx/onnx#3114]
# Bug Report

### Describe the bug

Build fails after updating to macOS Big Sur:

```
  Copying onnx.egg-info to build/bdist.macosx-10.15-x86_64/wheel/onnx-1.8.0-py3.8.egg-info
  running install_scripts
  [WARNING] This wheel needs a higher macOS version than the version your Python interpreter is compiled against.  To silence this warning, set MACOSX_DEPLOYMENT_TARGET to at least 11_0 or recreate these files with lower MACOSX_DEPLOYMENT_TARGET:
  build/bdist.macosx-10.15-x86_64/wheel/onnx/onnx_cpp2py_export.cpython-38-darwin.soTraceback (most recent call last):
    File "/Users/xxxxxxxxxx/env/onnx/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
```

```
AssertionError: would build wheel with unsupported tag ('cp38', 'cp38', 'macosx_11_0_x86_64')
```

### System information
- OS Platform and Distribution: macOS Big Sur 11.0.1 (20B29)
- Python version: 3.8
- Protobuf version: 3.13.0

### Reproduction instructions

```
#!/bin/bash
git clone --recursive https://github.com/onnx/onnx.git
python -m venv env
source ./venv/source/activate
export ONNX_ML=1
export ONNX_NAMESPACE=onnx
python -m pip install --upgrade pip
python -m pip install --use-feature=2020-resolver --upgrade protobuf
python -m pip install ./onnx
```

**Comment by jcwchen** [comment]
After this PR https://github.com/pypa/packaging/pull/319 merges, the new packaging PyPI package should help. Thank you for reporting this.

**Comment by lutzroeder** [comment]
Fixed in `packaging==20.5`.
